fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/CreatePolicyCommand.java

```diff
@@ -80,6 +88,26 @@ public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
 
     @Override
     public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        if (policyType == PolicyTypeEnum.DATA_MASK) {
+            if (user != null) {
+                user.analyze();
+                if (user.isRootUser() || user.isAdminUser()) {
+                    ErrorReport.reportAnalysisException(ErrorCode.ERR_TABLEACCESS_DENIED_ERROR, "CreatePolicyStmt",
+                            user.getQualifiedUser(), user.getHost(), nameParts.get(0));
+                }
+            }
+            // check auth
+            if (!Env.getCurrentEnv().getAccessManager()
+                    .checkGlobalPriv(ConnectContext.get(), PrivPredicate.GRANT)) {
+                ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR,
+                        PrivPredicate.GRANT.getPrivs().toString());
+            }
+            Policy policy = new DorisDataMaskPolicy(Env.getCurrentEnv().getNextId(), policyName, user, roleName,
+                    nameParts.get(0), nameParts.get(1), nameParts.get(2), nameParts.get(3), dataMaskType);
+            Env.getCurrentEnv().getPolicyMgr().createPolicy(policy, ifNotExists);
+            return;
+        }
+        ctx.getSessionVariable().enableFallbackToOriginalPlannerOnce();
```

Comment:
no need to enableFallbackToOriginalPlannerOnce, MustFallbackException will ignore enable_fallback variable and use legacy planner to execute it directly